contracts/Storage/BaseStorage.sol

```diff
@@ -4,107 +4,159 @@ import "./KeyValueStorage.sol";
 
 contract BaseStorage is KeyValueStorage {
 
+  /// @dev Only allow access if senderIsValid() is true
   modifier isAllowed {
     require(senderIsValid());
     _;
   }
 
   /**** Get Methods ***********/
 
+  /// @dev Get record for the key
```

Comment:
might want to specify for each of these the type, in case it generates docs someday. ex: `Get an address record for the given key`

also, I think `@param` should have a type specified as `bytes32` .. not sure what order it goes in

Reply:
Updated